### ISSUE TITLE
digraphs-1.6.1

### ISSUE THREAD

**Issue by sheerluck** [cschwan/sage-on-gentoo#728]
```diff
--- a/dev-gap/digraphs/digraphs-1.6.1.ebuild
+++ b/dev-gap/digraphs/digraphs-1.6.1.ebuild
@@ -35,7 +35,7 @@
 	# Unsetting ABI as gap use the variable internally.
 	econf \
 		--libdir="${ESYSROOT}/$(gap-pkg_path)/bin/$(gap-pkg_gaparch)" \
-		--with-gaproot="${EPREFIX}/usr/share/gap" \
+		--with-gaproot="$(gap_sysinfo_loc)" \
 		--with-external-planarity \
 		--with-external-bliss \
 		LIBS="-lgap" \
```

**Comment by kiwifb** [comment]
You are totally right.

**Comment by kiwifb** [comment]
fixed in 462b5b572c262b3d951167f62c1ecacbb9d71209
